content/1.2-rc/docs/installation/requirements.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Requirements"
+date: 2024-11-25
+---
+
+## Operators
+The orchestrator runtime/deployment is made of two main parts: `sonataflow operator` and `RHDH operator`  
+SonataFlow operator resource requirements are described [SonataFlow Installation Requirements](https://sonataflow.org/serverlessworkflow/latest/cloud/operator/install-serverless-operator.html#_prepare_a_minikube_instance)  
```

Comment:
the referenced links doesn't provide the resource requirements for OSL. it is for the minikube cluster.
this is meaningless for the admin that will install it on OCP.

since we don't have other data to provide here, at least let's refer to https://openshift-knative.github.io/docs/docs/latest/serverless-logic/getting-started/preparing-environment.html#proc-minimal-local-environment-setup
which is more formal and updated link, and also mention that this is for a local environment settings.

Reply:
fixed